README.md

```diff
@@ -66,65 +77,94 @@ Social Icons are optional. To show any of these icons, just provide the value in
   iconTitles = ["Twitter", "GitHub"]
 ```
 
-If any of these options are given, `hugo-theme-codex` will render the social icon in the footer, using the order specified in `iconTitles`.
+If any of these options are given, `hugo-theme-codex` will render the social
+icon in the footer, using the order specified in `iconTitles`.
 
-See the contents of the [example site](https://github.com/jakewies/hugo-theme-codex/tree/master/exampleSite) for more details.
+See the contents of the
+[example site](https://github.com/jakewies/hugo-theme-codex/tree/master/exampleSite)
+for more details.
 
 You can also create additional social icons by:
-1. Adding your own SVGs in `static/images/social/`, for example `static/images/social/reddit.svg`.
+
+1. Adding your own SVGs in `static/images/social/`, for example
+   `static/images/social/reddit.svg`.
 2. Modifying your site's config as follows:
+
    ```toml
    [params]
       # ...
       reddit = "<url to your reddit>"
-   
+
       iconTitles = [ "...", "Reddit"]
    ```
 
-Make sure that the icon title must match the icon's file name. If the title contains more than one word, say "My Awesome Site",
-you can use dash "-" for the icon name: `my-awesome-site.svg`. 
+Make sure that the icon title must match the icon's file name. If the title
+contains more than one word, say "My Awesome Site", you can use dash "-" for the
```

Comment:
Oh no wonder why so many lines were split into multiple ones, it is because of the 80-char limit. I believe nowadays most people use [120 chars](https://prettier.io/docs/en/options.html) as the line limit. 

I think it's good to increase the limit to 120, but actually it doesn't matter that much since it only affects the example contents inside this theme, so it's up to you.